src/index.js

```diff
@@ -0,0 +1,49 @@
+import './style.css';
+import UpdateStorage from './Modules/updateStorage.js';
+import { UpdateTask, saveUpdatedTask } from './Modules/edit.js';
+import {
+  displayTasks, deleteTask, add, gameChange, demolishAll,
+} from './functions.js';
+
+const taskInject = document.querySelector('.task-injector');
+const taskInput = document.querySelector('.task-input');
+const addBtn = document.querySelector('#add-sign');
+const completeAll = document.querySelector('.complete-delete');
+
+addBtn.addEventListener('click', () => add(taskInput, taskInject));
+taskInput.addEventListener('keypress', (event) => {
+  if (event.keyCode === 13) {
+    event.preventDefault();
+    add(taskInput, taskInject);
+  }
+});
+
+// Events
+document.addEventListener('click', (e) => deleteTask(e, taskInject));
+document.addEventListener('change', (e) => gameChange(e));
+completeAll.addEventListener('click', () => demolishAll(taskInject));
+
+taskInject.addEventListener('dblclick', (e) => {
+  if (e.target.classList.contains('dat-task')) {
+    UpdateTask(e);
+    const list = JSON.parse(localStorage.getItem('storedStTask')) || [];
```

Comment:
[ Optional ] Consider creating a separate function in a modules folder file for improved code reusability. Encapsulating common functionality in modular functions enhances maintainability and efficiency by enabling easy reuse across different files.